scripts/retryCommandExecutor.ps1

```diff
@@ -0,0 +1,110 @@
+# Copyright (C) Microsoft Corporation. All rights reserved.
+#This script runs a command with retry logic, handling specific errors, ignoring others, and considering timeout settings to ensure robust execution.
+<#
+ Example:
+ Execute-WithRetry -Command {
+     # Removing application fabric:/AAA
+     Remove-ServiceFabricApplication fabric:/AAA -ForceRemove -Force
+ } -Conditions @("FabricTransientException") -IgnoreConditions @("FabricElementNotFoundException")
+
+Define the conditions to handle
+$conditions = @("FabricTransientException", "AnotherTransientException")
```

Comment:
>FabricTransientException [](http://example.com/codeflow?start=17&length=24)

Frankly this should be the default, no point in repeating this in all callers.

Reply:
Done.